TransIT.DAL/Models/Entities/Manufacturer.cs

```diff
@@ -20,5 +21,7 @@ public class Manufacturer : IAuditableEntity, IBaseEntity
         public virtual User Create { get; set; }
 
         public virtual User Mod { get; set; }
+
+        public virtual ICollection<Part> Parts { get; set; }
```

Comment:
I'm not sure, but shouldn't you initialize `ICollection` in the constructor like in the `User/Post/Employee`?